app/models/user.rb

```diff
@@ -1,8 +1,27 @@
-# User
-class User < ActiveRecord::Base
+# frozen_string_literal: true
+
+class User < ApplicationRecord
   include ApiUser
+
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
Trackable moved into the comments - I'm guessing this was intentional / not from the auto-upgrade? 🤷‍♂

Reply:
It was not intentional (just regenerated files from devise) but it looks ok: we don't use the information provided by trackable...